src/main/java/homework1/DuplicateInArray.java

```diff
@@ -0,0 +1,21 @@
+package homework1;
```

Comment:
Please do not include previous tasks to the Pull Request. there are several ways to do so:
1. Change target branch to the task1 instead of main (code from the first task will be present in task2 branch, but will not be displayed as a diff)
2. Merge task1 branch into main (the same)
3. Create task2 branch from the main and not from the task1 branch (in such case - you will have separate branches for different tasks)

Reply:
thanks, done (actually you've done it during q&a session )